DigitalLibraryofTennessee/dltn_ingestion3_config_generator
# Add new sets for December ingest.

**Jira Issue**: [246](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/246)

## What does this Pull Request do?

Adds new sets for DPLA December ingest.

## What's new?

The following sets have been added:

- memphispublic_p16108coll5
- oakridge_p15388cdi
- oakridge_p16107coll3
- tsulibrarydigitalcollections
- tsupublicationallcatalgouesr0
- tsupublicationyearbooksr0

## How should this be tested?

Check to make sure all of the new sets are present and that the right value is included for these sets.
